doc/news/changes/minor/20200504SebastianStark

```diff
@@ -0,0 +1,3 @@
+Changed: Compute middle vertices during mesh refinement of Triangulation<2,3> using transfinite interpolation to be consistent with what happens to quads for Triangulation<3,3>
```

Comment:
Please break the line at around 80 characters.

Reply:
Thanks, well noted.